### ISSUE TITLE
Responsive/adaptive design?

### ISSUE THREAD

**Issue by loseth** [alan-luo/clutter#2]
Hi Alan!
I really love this theme, just what I need for myself, and it looks and feels gorgeous on a desktop.
But on mobile devices the breadcrumb menu is impossible to use because the headers of the pages overlap it.
I wish I had the knowledge to assist in fixing this, but this is really out of my league. Any chance that there will be a fix for this in the future?
Cheers
Tor

**Comment by faith-luo** [comment]
Yep, it's definitely fixable. I didn't really have responsive design in mind, since I originally designed this for myself, but I'm really glad that someone else has found a use for this theme! I'll definitely try to patch this in relatively soon, but I might be a bit busy over the next few weeks. Thanks for bringing this issue to mind.

**Comment by loseth** [comment]
Oh, that's great! No hurry though, Christmas and all coming up, family first. 
And if I find there is something I can contribute with I'll be sure to let you know. I'd love to learn PHP, YAML, and Twig, so if I find the time.... :-)

**Comment by faith-luo** [comment]
I'm finally back from traveling - I'll tackle this issue in the next 2 weeks. Thanks for your patience!

**Comment by ggolec** [comment]
Hi Alan, is there any progress on the subject? 
Would be great to have responsive design.

**Comment by flxzt** [comment]
I also would be very interested in this

**Comment by faith-luo** [comment]
Sorry everyone for the delay, I've been back at school. No definite timeframe on when I can get around to fixing this. However, I really appreciate all the feedback, and if there are any other features you guys would like to see in the next iteration, let me know.
